locator/topology.cc

```diff
@@ -11,6 +11,7 @@
 #include <seastar/core/on_internal_error.hh>
 #include <seastar/util/lazy.hh>
 #include <utility>
+#include <bit>
 
```

Comment:
There's a huge discrepancy between the hit in instruction count and the hit in throughput.

Reply:
What metrics do you consider having a huge discrepancy?
```
Before:
test                                               iterations      median         mad         min         max      allocs       tasks        inst      cycles
sort_by_proximity_topology.perf_sort_by_proximity      994665   996.540ns     0.900ns   995.639ns   998.420ns       0.000       0.000     13230.4      2982.9

After:
sort_by_proximity_topology.perf_sort_by_proximity     4260690   236.851ns     0.523ns   236.327ns   241.161ns       1.000       0.000      2847.9       710.0
```

Instructions: 13230.4 / 2847.9 = 4.65
Cycles: 2982.9 / 710.0 = 4.20
Iterations: 4260690. / 994665 = 4.28

Reply:
Ah, I see what you mean.  The last patch causes a much bigger hit in cycles than instructions.